src/Card-Container.css

```diff
@@ -0,0 +1,8 @@
+.card-container {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, 225px);
+  grid-gap: 20px;
+  justify-content: space-around;
+  padding: 25px;
```

Comment:
The padding and padding left is repetitive, can the padding-left be combined into the padding?

Reply:
Thank you Kaylee! got it fixed